site/app/views/admin/EmailRoomSeatingView.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace app\views\admin;
+
+use app\views\AbstractView;
+//TODO: add this as a general view in the future..
```

Comment:
what is this "TODO" supposed to mean? Try and rewrite this with the idea that someone not you might look at this and follow through on it.